packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -212,27 +224,42 @@ class MiniCart extends Component {
     };
 
     get miniCartInner() {
-        const { checkout, productList, props } = this;
+        const {
+            checkout,
+            productConfirm,
+            productList,
+            props,
+            state,
+            loading
+        } = this;
+
         const { classes, isCartEmpty, isMiniCartMaskOpen } = props;
+        const { isEditPanelOpen } = state;
 
         if (isCartEmpty) {
             return <EmptyMiniCart />;
         }
 
+        const footer = isEditPanelOpen ? productConfirm : checkout;
+
         const footerClassName = isMiniCartMaskOpen
             ? classes.footerMaskOpen
             : classes.footer;
 
         return (
             <Fragment>
-                <div className={classes.body}>{productList}</div>
-                <div className={footerClassName}>{checkout}</div>
+                <div className={classes.body}>
+                    {loading}
```

Comment:
Please rename this to something descriptive like `loadingIndicator`.

Reply:
Okay, `loading` renamed to `loadingIndicator`.